packages/veraswap-sdk/src/constants/tokens.ts

```diff
@@ -105,6 +117,7 @@ export const LOCAL_TOKENS: (HypERC20CollateralToken | HypERC20Token)[] = [
         token: mockTokens[1],
         connectionChainIds: [opChainA.id],
     }),
+    ...ethNativeTokens,
```

Comment:
This should technically be added to the `TESTNET_TOKENS` (though would need to rebase from `main` first for those to show up) so keep it as is for now.